PsychSourceGL/Cohorts/FTGLTextRenderer/OGLFT.cpp

```diff
@@ -3442,6 +3494,22 @@ QString Face::format_number ( const QString& format, double number ) { return( Q
     
     glEnd();
 
+    if (do_draw_underline_)
+    {
+        float undPos   = underline_position();
```

Comment:
Indentation should match the style done elsewhere in the file, and also be consistent.

Reply:
Sorry, used the shittiest of editors as my normal one had a licensing issue... Will fix throughout, and comment style also